scio-core/src/main/scala/com/spotify/scio/schemas/instances/JavaInstances.scala

```diff
@@ -69,4 +70,17 @@ trait JavaInstances {
   implicit def javaBeanSchema[T: IsJavaBean: ClassTag]: RawRecord[T] =
     RawRecord[T](new JavaBeanSchema())
 
+  implicit def javaEnumSchema[T <: java.lang.Enum[T] : ClassTag]: Schema[T] =
```

Comment:
TBH i think would prefer to use the straightforward way of using the `String` representation here. I see some problematic use case like when you are persist or have multiple pipelines running (i.e streaming) with "somehow" different Enum.

Reply:
I agree, I think it also helps for use cases when writing beam sql query? I.e. user wants to write enum name, not ordinal. If I understand how this whole thing works :)